nixos/tests/searx.nix

```diff
@@ -2,7 +2,7 @@ import ./make-test-python.nix ({ pkgs, ...} :
 
 {
   name = "searx";
-  meta = with pkgs.stdenv.lib.maintainers; {
+  meta = with pkgs.lib.maintainers; {
```

Comment:
Should we also do `pkgs.lib` -> `lib` in NixOS modules?

Reply:
Not sure, but I prefer not to do this in this commit anyway.